content/docs/started/getting-started-k8s.md

```diff
@@ -144,11 +144,342 @@ kubectl get cm dex -n kubeflow -o jsonpath='{.data.config\.yaml}' > dex-config.y
+            # Please note that if the bind password contains a `$`, it has to be saved in an
+            # environment variable which should be given as the value to `bindPW`.
+            bindDN: cn=admin,dc=arrikto,dc=com
+            bindPW: admin
+          
+            # The attribute to display in the provided password prompt. If unset, will
+            # display "Username"
+            usernamePrompt: Arrikto username
+          
+            # User search maps a username and password entered by a user to a LDAP entry.
+            userSearch:
+              # BaseDN to start the search from. It will translate to the query
+              # "(&(objectClass=person)(uid=<username>))".
+              baseDN: ou=People,dc=arrikto,dc=com
+              # Optional filter to apply when searching the directory.
+              filter: "(objectClass=inetOrgPerson)"
+          
+              # username attribute used for comparing user entries. This will be translated
+              # and combined with the other filter as "(<attr>=<username>)".
+              username: uid
+              # The following three fields are direct mappings of attributes on the user entry.
+              # String representation of the user.
+              idAttr: uid
+              # Required. Attribute to map to Email.
+              emailAttr: mail
+              # Maps to display name of users. No default value.
+              nameAttr: givenName
+          
+            # Group search queries for groups given a user entry.
+            groupSearch:
+              # BaseDN to start the search from. It will translate to the query
+              # "(&(objectClass=group)(member=<user uid>))".
+              baseDN: ou=Groups,dc=arrikto,dc=com
+              # Optional filter to apply when searching the directory.
+              filter: "(objectClass=groupOfNames)"
+          
+              # Following two fields are used to match a user to a group. It adds an additional
+              # requirement to the filter that an attribute in the group must match the user's
+              # attribute value.
+              userAttr: DN
+              groupAttr: member
+          
+              # Represents group name.
+              nameAttr: cn
+        {{< /highlight >}}
+        
+        </details>
+        
+    1. Append the LDAP config section to the dex config.
+        
+        {{< highlight bash >}}
+        cat dex-config.yaml dex-config-ldap-partial.yaml > dex-config-final.yaml
+        {{< /highlight >}}
+        
+    1. Apply the new config.
+        {{< highlight bash >}}
+        kubectl create configmap --from-file=dex-config-final.yaml -n kubeflow | kubectl apply -f -
+        {{< /highlight >}}
+        
+    1. Restart the Dex deployment, by doing one of the following:
+        * Force recreation, by deleting the Dex deployment's Pod(s).
+        * Trigger a rolling update, by adding/updating a label on the PodTemplate of the Dex deployment.
```

Comment:
Can we explain how to add/update a label, or link to docs that explain the process?